persist-cli-tests/src/test/resources/test-src/output/tool_test_init_9/persist/Persist.toml

```diff
@@ -0,0 +1,11 @@
+provider = "mysql"
+
+[database]
+host = "${perist_init_9.clients.host}"
+port = "${perist_init_9.clients.port}"
+user = "${perist_init_9.clients.user}"
+password = "${perist_init_9.clients.password}"
+database = "${perist_init_9.clients.database}"
+
+[shadowDatabase]
```

Comment:
Should we add this at this point itself? This is for future development isnt it?

Reply:
Yes, we don't need right now